vars/openShiftUtils.groovy

```diff
@@ -1,17 +1,35 @@
-// Helpers involving jenkins slaves running on openshift
+// Helpers for spinning up jenkins slaves running on OpenShift and other OpenShift utils
+
+
+private def getDefaultSlaveImage(String cloud) {
+    if (cloud.equals(pipelineVars.upshiftCloud)) return pipelineVars.centralCIjenkinsSlaveImage
+    else return pipelineVars.jenkinsSlaveImage
+}
+
+
+private def getDefaultSlaveNamespace(String cloud) {
+    if (cloud.equals(pipelineVars.upshiftCloud)) return pipelineVars.defaultUINameSpace
+    else return pipelineVars.defaultNameSpace
+}
+
+
+private def setDevPiEnvVars(String image, String cloud, Array envVars) {
+    // If using the IQE tests core image on the external OpenShift deployment, use the devpi
+    // server deployed in that external cluster
+    if (image == pipelineVars.iqeCoreImage && cloud == pipelineVars.defaultCloud) {
+        envVars.addAll([
+            envVar(key: 'PIP_TRUSTED_HOST', value: 'devpi.devpi.svc'),
+            envVar(key: 'PIP_INDEX_URL', value: 'http://devpi.devpi.svc:3141/root/psav'),
+        ])
+    }
+}
```

Comment:
I hope it will go once we deploy our private devpi server that will be accessible for both clusters.